pkg/flowcli/project/project.go

```diff
@@ -62,7 +62,7 @@ func Load(configFilePath []string) (*Project, error) {
 			return nil, err
 		}
 
-		return nil, fmt.Errorf("failed to open project configuration: %s", configFilePath)
+		return nil, err
```

Comment:
Could we add a `config.ErrOutdatedFormat` error that gets caught here?

Then we could log a message that includes a link to the docs for the new format